guidelines/monai-application-package.md

```diff
@@ -1,498 +1,588 @@
-  - [Manifest Export](#manifest-export)
+- [Supplemental Application Files](#supplemental-application-files)
+  - [Container Behavior and Interaction](#container-behavior-and-interaction)
   - [Table of Important Paths](#table-of-important-paths)
-- [Package Layout Diagram](#layout-diagram)
+- [Package Layout Diagram](#package-layout-diagram)
 
+<!-- /code_chunk_output -->
 
 ## Overview
 
-This proposal documents the specification of the initial version of the MONAI Application Package (MAP).
-
+This proposal documents the specification of the MONAI Application Package (MAP).
 
 ### Goal
 
-The goal of this proposal is to provide the structure of a MAP, define the purpose of a MAP and how it can be
+The goal of this proposal is to provide the structure of a MAP, define the purpose of a MAP, and how it can be
 interacted with, and the required and optional components of a MAP.
 
+### Assumptions, Constraints, Dependencies
 
-### Assumptions
-
-The following are a set of assumptions related to MAP execution, inspection, and general usage.
+The following is a set of assumptions about MAP execution, inspection, and general usage.
 
-- Containerized applications will be based on Linux x86_64 (AMD64).
+- Containerized applications will be based on Linux x64 (AMD64) and/or ARM64 (aarch64).
 
-- Containerized application's host environment will be based on Linux x86_64 (AMD64) with container support.
+- Containerized application's host environment will be based on Linux x64 (AMD64) and/or ARM64 (aarch64) with container support.
 
-- Developers expect local execution of their applications to behave identically to execution of the containerized version.
+- Developers expect the local execution of their applications to behave identically to the execution of the containerized version.
 
 - Developers expect local execution of their containerized applications to behave identically to execution in deployment.
 
 - Developers and operations engineers want the application packages to be self-describing.
 
-- Application packages might not be developed using the MONAI Application SDK.
+- Applications might not be developed using the Holoscan SDK or an SDK based on it, e.g., MONAI Deploy App SDK.
+
+- MONAI Application Package may not be created using the Holoscan SDK nor the MONAI Deploy App SDK.
 
-- Pre-existing, containerized applications will need to be "converted" into MONAI Application Packages.
+- Pre-existing, containerized applications must be "converted" into MONAI Application Packages.
 
-- Application packages will be deployed in a variety of environments.
+- The scalability of a multi-fragment application based on Holoscan SDK is outside the scope of this document.
+
+- A MONAI Application Package may contain a classical application (non-fragment based), a single-fragment application, or a multi-fragment application.
+
+- Application packages are expected to be deployed in a variety of environments.
```

Comment:
Would it make sense to simply say:
"Application packages are expected to be deployed in one of the supported environments. See [MONAI Operating Environments](monai-operating-environments.md) for additional information"